base/applications/rapps/settingsdlg.cpp

```diff
@@ -243,20 +243,20 @@ namespace
                 }
 
 
-                if(NewSettingsInfo.bUseSource && !IsUrlValid(szSource.GetString()))
+                if(NewSettingsInfo.bUseSource && !IsUrlValid(szSource))
                 {
-                    ATL::CStringW szMsgText;
+                    CStringW szMsgText;
                     szMsgText.LoadStringW(IDS_URL_INVALID);
 
-                    MessageBoxW(hDlg, szMsgText.GetString(), NULL, 0);
+                    MessageBoxW(hDlg, szMsgText, NULL, 0);
                     SetFocus(GetDlgItem(hDlg, IDC_SOURCE_URL));
                     break;
                 }
                 else
                 {
-                    ATL::CStringW::CopyChars(NewSettingsInfo.szSourceURL,
+                    CStringW::CopyChars(NewSettingsInfo.szSourceURL,
                                          _countof(NewSettingsInfo.szSourceURL),
-                                         szSource.GetString(),
+                                         szSource,
                                          szSource.GetLength() + 1);
```

Comment:
Those three lines above may require one less spacing (to align with the 1st parameter).